Resources/Prototypes/Entities/Mobs/NPCs/carp.yml

```diff
@@ -43,9 +43,8 @@
       crit: crit
       dead: dead
   - type: Butcherable
-    # TODO: CrapMeat or FishMeat # - 2022-02-17 LMAO crap meat
```

Comment:
leave the comment its funny!!!!!

Reply:
and soul etc etc

Reply:
My friendship with crapmeat is over.
Meatfish is my new best friend.